io.catenax.user_estimated_loading/1.0.0/UserEstimatedLoading.ttl

```diff
@@ -0,0 +1,244 @@
+
+:HistoryProfilesEnity a samm:Entity ;
+   samm:preferredName "History profiles entity"@en ;
+   samm:description "collection of previous usage profiles "@en ;
+   samm:properties ( :percentage [ samm:property :trailerUse; samm:optional true ] [ samm:property :cityDriving; samm:optional true ] :drivingStyle :hilliness :curviness ) .
+
+:FutureListEntity a samm:Entity ;
+   samm:preferredName "future list entity"@en ;
+   samm:description "List of future scenarios"@en ;
+   samm:properties ( :futureScenario :futureScenarioId ) .
+
+:mileage a samm:Property ;
+   samm:preferredName "mileage @en"@en ;
+   samm:description "distance the vehicle has travelled in kilometre"@en ;
+   samm:characteristic :Mileage ;
+   samm:exampleValue "123456"^^xsd:positiveInteger .
+
+:ageInMonth a samm:Property ;
+   samm:preferredName "vehicle age @en"@en ;
+   samm:description "age of the vehicle in month"@en ;
+   samm:characteristic :AgeInMonths ;
+   samm:exampleValue "42"^^xsd:nonNegativeInteger .
+
+:vehicleType a samm:Property ;
+   samm:preferredName "vehicle variant"@en ;
+   samm:description "Vehicle variant - body shapes like SUV, Van, Limousine"@en ;
+   samm:characteristic :KbaVariant ;
+   samm:exampleValue "SUV" .
+
+:numberOfDriveAxle a samm:Property ;
+   samm:preferredName "number of drive axles"@en ;
+   samm:description "identification of 4WD, default 1: 2WD"@en ;
+   samm:characteristic :NumberOfDriveAxle ;
+   samm:exampleValue "2"^^xsd:positiveInteger .
+
+:energySourceCode a samm:Property ;
+   samm:preferredName "energy source  code"@en ;
+   samm:description "the code for the energy source respectively fuel type of the vehicle according to KBA list"@en ;
+   samm:characteristic :EnergySource ;
+   samm:exampleValue "\"0002\"" .
+
+:powerKW a samm:Property ;
+   samm:preferredName "power @en"@en ;
+   samm:description "engine power is the power that an engine can put out in kW"@en ;
+   samm:characteristic :EnginePowerKW ;
+   samm:exampleValue 145 .
+
+:grossVehicleWeightKG a samm:Property ;
+   samm:preferredName "gross vehicle weight"@en ;
+   samm:description "technical  permissable maximum weight of the vehicle in kg"@en ;
+   samm:characteristic :Weight .
+
+:percentage a samm:Property ;
+   samm:preferredName "weightfactor in percantage"@en ;
+   samm:description "Weighfactor of the different profiles, a user has. Different profiles indicates different user as well as different usages of like profiles for daily work routes or for caravan holiday."@en ;
+   samm:characteristic :Percentage ;
+   samm:exampleValue "23.5"^^xsd:decimal .
+
+:trailerUse a samm:Property ;
+   samm:preferredName "traile use switch"@en ;
+   samm:description "Usage of trailer like caravan. "@en ;
+   samm:characteristic :Boolean ;
+   samm:exampleValue true .

```

Comment:
i assume you need to qualify with type info as well. ie "true"^^xsd.boolean also for all other occasions